### PR TITLE
Settings system UI

### PR DESCRIPTION
**Pull request by DavidCramer**

### CODE REVIEW

**Review by pereirinha — COMMENTED** [review]
@DavidCramer 

I left you a few comments.

File: php/class-settings.php [hunk 1]
```diff
@@ -97,13 +111,29 @@ public function render() {
...
+	public function bind_footer() {
+		ob_start();
+		add_action( 'admin_footer', array( $this, 'render_footer' ) );
+	}
+
+	/**
+	 * Render the footer in admin page.
+	 */
+	public function render_footer() {
+		ob_get_clean();
```
[the remark this answers is not in the document]
Comment (DavidCramer) [reply php/class-settings.php:133]: @pereirinha - I'll leave this for you to think of a solution for now.

**Comment by DavidCramer** [comment]
@pereirinha - issues addressed, and left a comment where not.

**Review by pereirinha — APPROVED** [review]
I'll merge as is, and we can circle back into the footer question at a later stage.

**Pull request merged**